contracts/src/v0.8/operatorforwarder/dev/AuthorizedForwarder.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {ConfirmedOwnerWithProposal} from "../../shared/access/ConfirmedOwnerWithProposal.sol";
+import {AuthorizedReceiver} from "./AuthorizedReceiver.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+// solhint-disable custom-errors
+contract AuthorizedForwarder is ConfirmedOwnerWithProposal, AuthorizedReceiver {
+  using Address for address;
+
+  // solhint-disable-next-line chainlink-solidity/prefix-immutable-variables-with-i
+  address public immutable getChainlinkToken;
+
+  event OwnershipTransferRequestedWithMessage(address indexed from, address indexed to, bytes message);
+
+  constructor(
+    address link,
+    address owner,
+    address recipient,
+    bytes memory message
+  ) ConfirmedOwnerWithProposal(owner, recipient) {
+    require(link != address(0), "Link token cannot be a zero address");
+    getChainlinkToken = link;
+    if (recipient != address(0)) {
+      emit OwnershipTransferRequestedWithMessage(owner, recipient, message);
+    }
+  }
+
+  /**
+   * @notice The type and version of this contract
+   * @return Type and version string
+   */
+  function typeAndVersion() external pure virtual returns (string memory) {
```

Comment:
Prefer this format 

  // solhint-disable-next-line chainlink-solidity/all-caps-constant-storage-variables
  string public constant override typeAndVersion = "AuthorizedForwarder 1.0.0";